src/main/java/com/questhelper/spells/StandardSpellBuilder.java

```diff
@@ -0,0 +1,88 @@
+/*
+ *
+ *  * Copyright (c) 2021, Senmori
+ *  * All rights reserved.
+ *  *
+ *  * Redistribution and use in source and binary forms, with or without
+ *  * modification, are permitted provided that the following conditions are met:
+ *  *
+ *  * 1. Redistributions of source code must retain the above copyright notice, this
+ *  *    list of conditions and the following disclaimer.
+ *  * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *  *    this list of conditions and the following disclaimer in the documentation
+ *  *    and/or other materials provided with the distribution.
+ *  *
+ *  * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ *  * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ *  * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ *  * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ *  * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ *  * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ *  * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ *  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+package com.questhelper.spells;
+
+import com.questhelper.requirements.ItemRequirement;
+import com.questhelper.requirements.ItemRequirements;
+import com.questhelper.requirements.util.LogicType;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Locale;
+import org.apache.commons.text.WordUtils;
+
+public class StandardSpellBuilder
+{
+	private final Spell spell;
+	private final List<ItemRequirement> requirementList = new LinkedList<>();
+
+	private StandardSpellBuilder(Spell spell)
+	{
+		this.spell = spell;
+	}
+
+	public static StandardSpellBuilder builder(Spell spell)
+	{
+		return new StandardSpellBuilder(spell);
+	}
+
+	public StandardSpellBuilder rune(int quantity, Rune rune)
```

Comment:
I decided on `(int, Rune)` because it reads easier. 
For example: 2 air runes, 2 water runes
That's a bit easier to read than:
air runes x 2, water runes x 2

This especially helps in `Rune` where it's very dense.